tools/buildInstaller.js

```diff
@@ -51,39 +85,39 @@ if (isDarwin) {
     raiseError('IDENTIFIER needs to be set to the certificate organization')
   }
 
-  const wvBundle = buildDir + '/Brave.app/Contents/Frameworks/Brave Framework.framework/Brave Framework'
-  const wvBundleDir = buildDir + '/Brave.app/Contents/Frameworks/Widevine Resources.bundle'
+  const wvBundle = buildDir + `/${appName}.app/Contents/Frameworks/Brave Framework.framework/Brave Framework`
+  const wvBundleDir = buildDir + `/${appName}.app/Contents/Frameworks/Widevine Resources.bundle`
   const wvContents = wvBundleDir + '/Contents'
   const wvResources = wvContents + '/Resources'
   const wvBundleSig = wvResources + '/Brave Framework.sig'
-  const wvPlugin = buildDir + '/Brave.app/Contents/Frameworks/Brave Framework.framework/Libraries/WidevineCdm/_platform_specific/mac_x64/widevinecdmadapter.plugin'
+  const wvPlugin = buildDir + `/${appName}.app/Contents/Frameworks/Brave Framework.framework/Libraries/WidevineCdm/_platform_specific/mac_x64/widevinecdmadapter.plugin`
   cmds = [
     // Remove old
-    'rm -f ' + outDir + '/Brave.dmg',
+    'rm -f ' + outDir + `/${appName}.dmg`,
 
     // sign for widevine
     'mkdir -p "' + wvResources + '"',
-    'cp ' + buildDir + '/Brave.app/Contents/Info.plist "' + wvContents + '"',
+    'cp ' + buildDir + `/${appName}.app/Contents/Info.plist "` + wvContents + '"',
     'codesign --deep --force --strict --verbose --sign $IDENTIFIER "' + wvBundle + '"',
     'codesign --deep --force --strict --verbose --sign $IDENTIFIER "' + wvPlugin + '"',
     'python tools/signature_generator.py --input_file "' + wvBundle + '" --output_file "' + wvBundleSig + '" --flag 1',
     'python tools/signature_generator.py --input_file "' + wvPlugin + '"',
 
     // Sign it
-    'cd ' + buildDir + '/Brave.app/Contents/Frameworks',
+    'cd ' + buildDir + `/${appName}.app/Contents/Frameworks`,
     'codesign --deep --force --strict --verbose --sign $IDENTIFIER *',
     'cd ../../..',
-    'codesign --deep --force --strict --verbose --sign $IDENTIFIER Brave.app/',
+    `codesign --deep --force --strict --verbose --sign $IDENTIFIER ${appName}.app/`,
 
     // Package it into a dmg
     'cd ..',
     'build ' +
-      '--prepackaged="' + buildDir + '/Brave.app" ' +
+      '--prepackaged="' + buildDir + `/${appName}.app" ` +
```

Comment:
maybe we should change to also use template literal syntax too- so like:
 `--prepackaged="${buildDir}/${appName}.app" ` +